src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.ts

```diff
@@ -312,6 +319,39 @@ export class BuilderNavbarComponent implements OnDestroy {
     this.store.cancelSubmission();
   }
 
+  /**
+   * Create a new changelog for the active learning object
+   */
+  updateChangelog() {
```

Comment:
If this function creates a new changelog, why is it called `updateChangelog`?
![image](https://user-images.githubusercontent.com/7431390/55911113-f4709700-5bad-11e9-8699-f258d74a4c30.png)

Reply:
Now says createChangelog()